src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const images = [...document.querySelectorAll('a')];
+
+images.forEach((image) => {
+  image.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const link = e.target.closest('img');
```

Comment:
The `link` variable is attempting to find the closest 'img' element, but since the event is attached to anchor tags, this will not work as intended. You should directly use `e.currentTarget` to get the anchor element.